tasks/run.js

```diff
@@ -39,3 +24,25 @@ async function runTask (name, args = []) {
     task.on('close', resolve)
   })
 }
+
+/**
+ * Creates a Gulp task for runTask()
+ *
+ * @param {string} name - npm script name
+ * @param {string[]} [args] - npm script arguments
+ * @returns {() => Promise<number>} Exit code
+ */
+const runTask = (name, args = []) => {
+  const script = `${name} ${args.join(' ')}`.trim()
+
+  // Create named runner
+  // https://gulpjs.com/docs/en/api/task#task-metadata
+  const runner = () => task(name, args)
```

Comment:
I think there's a bit of naming collision there as this is not the `task` function from Gulp, but our own from right above. We may want to rename that `taskForNpmScript` for clarification, especially as it doesn't have the same signature as Gulp's `task`.

Reply:
Since Gulp v4 tasks can be "[a stream, promise, event emitter, child process, or observable](https://gulpjs.com/docs/en/getting-started/creating-tasks)"

I've flipped it the other way around (function that returns `Promise<number>`)

```suggestion
  const task = () => npmScript(name, args)
```